src/client/Client.cc

```diff
@@ -2510,9 +2510,7 @@ void Client::handle_osd_map(const MConstRef<MOSDMap>& m)
     objecter->maybe_request_map();
   }
 
-  if (objecter->osdmap_full_flag()) {
```

Comment:
@dzafman i also tried to remove the handling of osdmap full/nearfull on the client side in #30614, but i was stopped by Sage. because the new client still needs to be backward compatible with old cluster which still sends the full/nearfull in osdmaps.